modules/storages/app/views/storages/admin/storages/show_oauth_application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!-- Standard Ruby view, please see the controller for comments -->
 <% html_title t(:label_administration), t("project_module_storages"), @object.name, "#{t("storages.provider_types.#{@object.provider_type}.name")} #{t("storages.label_oauth_application_details")}" %>
-<% local_assigns[:additional_breadcrumb] = "#{t("storages.provider_types.#{@object.provider_type}.name")} #{t("storages.label_oauth_application_details")}" %>
-<%= toolbar title: "#{t("storages.provider_types.#{@object.provider_type}.name")} #{t("storages.label_oauth_application_details")}" %>
+<% local_assigns[:additional_breadcrumb] = %W[#{@object.name} #{t("storages.label_oauth_breadcrumb.openproject")}] %>
```

Comment:
Don't know the requirements here, is it correct to always refer to `openproject`?

Reply:
Yes, this view will always show a created OAuth application which is created in OpenProject. Only the preceding view can be different depending on the type of the attached storage.